### PR TITLE
daemon: Create cockpit-udisks

### PR DESCRIPTION
**Pull request by petervo**
Create new storage only daemon so cockpitd can run on systems without udisk and storaged support.

### CODE REVIEW

**Comment by petervo** [comment]
Renamed to cockpit-udisks

File: tools/cockpit.spec [hunk 1]
```diff
@@ -253,6 +262,11 @@ rm -rf %{buildroot}/debug
 %{_datadir}/dbus-1/services/com.redhat.Cockpit.service
 %{_libexecdir}/cockpitd
 
+%files udisks
+%{_libexecdir}/cockpit-udisks
+%{_datadir}/dbus-1/services/com.redhat.Cockpit.Storage.service
+%doc %{_mandir}/man8/cockpit-udisks.8.gz
```
Comment (stefwalter) [inline tools/cockpit.spec]: I think this is an implementation detail, and doesn't need a man page. Similar to how 'cockpit-pcp' doesn't have a man page.

**Comment by petervo** [comment]
Removed man entries

**Comment by stefwalter** [comment]
Alternate pull request #2078

**Comment by mvollmer** [comment]
> Alternate pull request #2078

We decided to merge #2078 instead.

**Pull request closed without merging**